test/integration/wyatt_test.rb

```diff
@@ -0,0 +1,10 @@
+require 'test_helper'
+
+class WyattTest < ActionDispatch::IntegrationTest
```

Comment:
Defining the WyattTest class, which inherits from ActionDispatch::IntegrationTest.